### ISSUE TITLE
Running tasks in parallel

### ISSUE THREAD

**Issue by rbaral** [girder/girder_worker#51]
Hi,

Does the girder worker support parallel execution of tasks inside a workflow.
For instance, lets say I have a workflow having tasks T1,T2,T3,T4... Is there a way I can run tasks T2 and T3 in parallel after task T1 is completed?

Thanks.

**Comment by jeffbaumes** [comment]
This feature is not implemented as of yet in girder worker directly. To do this you would need to start the `girder-worker` server and send the various tasks through celery manually (calling `run`). Celery supports parallelism with (groups)[http://docs.celeryproject.org/en/latest/userguide/canvas.html#groups].

I'd like to keep this issue open to track this if we want to implement a better solution that girder worker handles directly.

**Comment by rbaral** [comment]
Yes, celery does provide the parallel execution, but I need to run some tasks of the workflow itself to run in parallel. I can run the whole workflow instances in parallel using celery or raw python parallelism. The parallelism within the workflow is the good to have feature and also highlighted in the workflow patterns [http://www.workflowpatterns.com/patterns/control/]

Thanks.

**Comment by jeffbaumes** [comment]
Agreed, it would be a great feature to have.
